app/models/review.rb

```diff
@@ -1,5 +1,5 @@
 class Review < ApplicationRecord
-  validates_presence_of :rating, numericality: { greater_than: 0, less_than_or_equal_to: 10 }
+  validates :rating, presence: true, numericality: { greater_than: 0, less_than_or_equal_to: 10 }
```

Comment:
FYI you don't need to validate presence here since numericality is doing it by default!